packages/design-system/.storybook/preview-head.html

```diff
@@ -3,7 +3,6 @@
 	href="https://fonts.googleapis.com/css2?family=Gelasio:ital,wght@0,400;0,500;0,600;0,700;1,400;1,500;1,600;1,700&display=swap"
 	rel="stylesheet"
 />
-<link id="bootstrap-theme" href="https://talend.surge.sh/theme/bootstrap.css" rel="stylesheet" />
```

Comment:
provided by shared storybook config

Reply:
I think you have broken the toggle switch